addons/calendar_sms/models/calendar_alarm.py

```diff
@@ -23,3 +25,11 @@ def _compute_sms_template_id(self):
                 alarm.sms_template_id = self.env['ir.model.data']._xmlid_to_res_id('calendar_sms.sms_template_data_calendar_reminder')
             elif alarm.alarm_type != 'sms' or not alarm.sms_template_id:
                 alarm.sms_template_id = False
+
+    @api.onchange('duration', 'interval', 'alarm_type', 'do_not_remind_responsible')
```

Comment:
note for reviewer -> inheriting an onchange, should I only add the new field, or like here, specify all of them?